README.md

```diff
@@ -142,26 +118,18 @@ publishing steps into one workflow.
 
 Ask @chriddyp to get NPM / PyPi package publishing access.
 
-1. Preparing to publish to NPM
-
-        # Bump the package version
-        $ npm version major|minor|patch
-
-        # Push branch and tags to repo
-        $ git push --follow-tags
-
-2. Preparing to publish to PyPi
+1 - Update `version.py`
```

Comment:
we're using semantic versioning in this repo right? (cc @chriddyp )
If that's the case, we may want to link to docs for people not super familiar with how to bump the versioning https://semver.org/